components/movementsensor/dualgps/dualgps.go

```diff
@@ -264,7 +264,12 @@ func (dg *dualGPS) Orientation(ctx context.Context, extra map[string]interface{}
 }
 
 func (dg *dualGPS) Accuracy(ctx context.Context, extra map[string]interface{}) (*movementsensor.Accuracy, error) {
-	return nil, movementsensor.ErrMethodUnimplementedAccuracy
+	// RSDK-6389: for this driver, find the highest value VDOP and HDOP to show the worst accuracy in position
+	// check the fix and return the compass error using a calculation based on the fix and variation
+	// of the two positions.
+	// return the NMEA Fix that appears from either gps in this order: (worst) 0, 1, 2, 5, 4 (best), or -1 if
+	// it is not any of these (other nmea GGA fixes indicate simulated or deadreckoning gps devices)
+	return movementsensor.UnimplementedAccuracies()
```

Comment:
Same confusion re: comment not matching code here. If a comment is meant as a TODO, I'd start it off with `// TODO[RSDK-6389]:` to make that clear.